src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,17 @@
-export const TodoList = () => {};
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
```

Comment:
Similar names are okay for a small project, but on large projects we will have many interfaces and types, so it is better to give more detailed names, for example `TodoListProps` or `TodoListType`.
This applies to all code.